.gitignore

```diff
@@ -348,3 +348,5 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+.DS_Store
```

Comment:
Why do we need this file?

Reply:
We don't..that's why it's added in gitignore.
this file is a mac thing that helps specify how to display files and folders.
Let me remove it from the commit actually.

Reply:
Removed file from commit.